doc/user/layouts/partials/create-source/connector/s3/details.html

```diff
@@ -1,22 +1,76 @@
 ### S3 source details
 
-The S3 source is designed to ingest a large volume of static data from S3.
+The S3 source is designed to ingest a large volume of static data from [AWS's Simple Storage
+Service][aws-s3]. S3 is Amazon Web Services' cloud object store. It is important to note that an
+object store behaves differently than a file system does, and that those differences affect how
```

Comment:
You can simplify the beginning by combining the first two sentences: "The S2 source is designed to ingest a large volume of static data from aws-s3, Amazon Web Services' cloud object store.